### ISSUE TITLE
Replies accidentally sent to wrong person

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#116]
```
Please introduce a 10 second delay between receiving a message, and "reply:" 
acting on that later message, to avoid this scenario:

1) Jane sends me a message 
2) I type "reply:I can't wait for you to come over tonight too" 
3) Just before I press enter, Suzy sends me a message 
4) I press enter on my reply to Jane 
5) The reply goes to Suzy! 

```

Original issue reported on code.google.com by `earthlin...@googlemail.com` on 5 Mar 2011 at 1:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 6 Mar 2011 at 9:22
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 7 Mar 2011 at 1:47
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.0
```

Original comment by `Florent....@gmail.com` on 8 Mar 2011 at 7:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
